R/tt_toString.R

```diff
@@ -274,8 +273,26 @@ setMethod("matrix_form", "VTableTree",
                                body_ref_strs)),
                    nrow = nrow(body),
                    ncol = ncol(body))
+    # Solve \n in titles
+    if (any(grepl("\n", all_titles(obj)))) {
+        if (any(grepl("\n", main_title(obj)))) {
+            tmp_title_vec <- .quick_handle_nl(main_title(obj))
+            main_title(obj) <- tmp_title_vec[1]
+            subtitles(obj) <- c(tmp_title_vec[-1], .quick_handle_nl(subtitles(obj)))
+        } else {
+            subtitles(obj) <- .quick_handle_nl(subtitles(obj))
+        }
+    }
+    
+    # Solve \n in footers
+    main_footer(obj) <- .quick_handle_nl(main_footer(obj))
+    prov_footer(obj) <- .quick_handle_nl(prov_footer(obj))
+    
+    # xxx \n in page titles are not working atm (I think)
+    # ref_fnotes <- strsplit(get_formatted_fnotes(obj), "\n", fixed = TRUE)
```

Comment:
This would be a solution that keeps pagination correct but it is problematic in multiple cases as it would print the form of a vector (with `c()`, etc)